backend/src/converter/endpoints.ts

```diff
@@ -1,41 +1,48 @@
 import { validateConverter } from "./service";
-import { getCollections } from "../app";
+import { getCollections } from "../db";
 import { Request, Response } from "express";
 import { ObjectId } from "mongodb";
 
 export const getAllConverters = async (_req: Request, res: Response) => {
     const { converters } = getCollections();
-    try {
-        const result = await converters.find({}).toArray();
+    let result;
 
-        if (!result || result.length === 0) {
-            res.status(204).send();
-        } else {
-            res.status(200).json(result);
-        }
+    try {
+        result = await converters.find({}).toArray();
     } catch (err) {
-        res.status(500).json(err);
+        res.status(500).json({ error: "Database error" });
+        return;
+    }
+
+    if (!result || result.length === 0) {
+        res.status(404).json({ error: "No converters found" });
```

Comment:
404 odpowiada za trochę coś innego https://stackoverflow.com/questions/34312023/http-get-request-status-204-vs-404

Reply:
Właściwie to nie wiem czy powinniśmy wgl tutaj zwracać jakiś błąd? W ogólnym przypadku pusta tablica może być poprawną odpowiedzią na jakieś żądanie. Pytanie, czy kojarzysz, dlaczego zwracaliśmy wcześniej taki błąd w przypadku pustych tablic? Ja byłbym za tym, żeby w przypadku pustej tablicy zwracać normalnie status 200, ale to też do sprawdzenia czy nie spowoduje to jakiś błędów na frontendzie.

Reply:
dobra może faktycznie miałem tu na 200 zmienić i pomyliłem, bo pamiętam tylko że 204 było źle, ale ma to więcej sensu

Reply:
No to daj na 200 i powinno być git